src/curves.rs

```diff
@@ -96,6 +96,25 @@ macro_rules! new_curve_impl {
             }
         }
 
+        impl group::WnafGroup for $name {
+            fn recommended_wnaf_for_num_scalars(num_scalars: usize) -> usize {
+                // Copied from bls12_381::g1, should be updated.
+                const RECOMMENDATIONS: [usize; 12] =
+                    [1, 3, 7, 20, 43, 120, 273, 563, 1630, 3128, 7933, 62569];
+
+                let mut ret = 4;
```

Comment:
@ebfull how did you go about generating these recommendations for BLS12-381 in the first place? It would be nice if `group` could provide some way to derive these, or at least document how to go about it.

Reply:
Note that these values only matter for fixed-base usage; we hard-code the window size for variable-base w-NAF (initiated via `Wnaf::new().scalar(..)`).